src/pages/Home/Home.scss

```diff
@@ -1,91 +1,79 @@
-.bodyPage{
-  font-family: "SF Pro Text","Helvetica Neue",-apple-system,BlinkMacSystemFont,"Helvetica","Segoe UI","Arial",sans-serif;
+.bodyPage {
```

Comment:
Gosto de aplicar estilos usando o Styled Components. No entanto o Sass já é bem melhor do que o css puro o/

Reply:
Podemos levantar esse assunto, tem vários pontos a serem considerados quando falamos de css in js vs css preprocessor 😄